applications/OptimizationApplication/python_scripts/algorithms/algorithm.py

```diff
@@ -68,3 +69,8 @@ def CallOnAllProcesses(self, process_types: 'list[str]', *args, **kwargs):
         for process_type in process_types:
             CallOnAll(self._optimization_problem.GetListOfProcesses(process_type), *args, **kwargs)
 
+    def _InitializeIteration(self) -> None:
+        self.CallOnAllProcesses(self._optimization_problem.GetAvailableProcessTypes(), Kratos.Process.ExecuteInitializeSolutionStep)
```

Comment:
isnt this the same as `map(Kratos.Process.ExecuteInitializeSolutionStep, self._optimization_problem.GetAvailableProcessTypes())` ?

Did I make this commend before? :sweat_smile:

Reply:
Yes you have. I forgot to change it. :/

Reply:
done :)